packages/editor/src/components/header/index.js

```diff
@@ -73,6 +75,8 @@ function Header( {
 			isPublishSidebarOpened: _isPublishSidebarOpened(),
 			showIconLabels: getPreference( 'core', 'showIconLabels' ),
 			hasFixedToolbar: getPreference( 'core', 'fixedToolbar' ),
+			isBlockSelected:
```

Comment:
Not a blocker at all, but a tiny naming nit: `isBlockSelected` usually makes me think of is a particular block selected. I see in a couple of other places we've used the name `hasBlockSelection` for this kind of check so that it communicates that it's about whether any block is selected, rather than a particular one.

I doubt it'll cause any confusion here using `isBlockSelected`, so don't worry about changing it if you think it's a non-issue!

Reply:
Agreed, I've changed it 👍